packages/frontend-app/src/app/components/Card/index.scss

```
@@ -0,0 +1,20 @@
+@import "tailwindcss/components";
+@import "tailwindcss/base";
+@import "tailwindcss/utilities";
+
+.card__container {
+  @apply flex flex-col cursor-pointer w-144 lg:w-156 xl:w-214 2xl:w-250;
+}
+
+.cover__image {
+  @apply rounded-md w-full max-h-144 lg:max-h-156 xl:max-h-214 2xl:max-h-250;
+}
+
+.overlay {
+  @apply absolute bottom-0 left-0 w-full h-full grid content-end  opacity-0 hover:opacity-100 hover:ease-in hover:duration-300;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+
+.overlay-content {
+  @apply flex justify-between px-4 pb-4;
+}
```

You want to make sure you avoid global name clashes so please avoid doing this. When you do this, it might clash with an overlay for another component. You wanna have these classes scoped within the parent using scss.

- Also for transitions, you want the transition property not on the pseudo-classes but on the element itself.

- Also no need to have an extra `div` with the class name `overlay-content`. You can do all with the overlay component. Check out the new classes I added to the `overlay` component

- Please create a bg-blackAlpha30 in the tailwing config file. You should already see bg-blackAlpha45 in the file. Add it close to that

```suggestion
.card__container {
   & .overlay {
  @apply absolute bottom-0 left-0 w-full h-full bg-blackAlpha30
  @apply flex items-end justify-between p-4 
  @apply opacity-0 hover:opacity-100 ease-in duration-300;
}
}
```